frontend/ui/translator.lua

```diff
@@ -377,6 +387,20 @@ function Translator:loadPage(text, target_lang, source_lang)
     local query = ""
     self.trans_params.tl = target_lang
     self.trans_params.sl = source_lang
+    if G_reader_settings:isTrue("translator_from_romanizations") then
+       table.insert(self.trans_params.dt, "rm")
+    else
+       local rmIndex = nil
+       for i, v in pairs(self.trans_params.dt) do
+          if v == "rm" then
+             rmIndex = i
+             break
+          end
+       end
```

Comment:
We have
https://github.com/koreader/koreader/blob/8297d17e54374fcb568e14b03a3a9ca94aac577e/frontend/util.lua#L323